Symfony migration from 3.1 to 4.4

briandidierjean opened the pull request:
This migrates from Symfony 3.1 to 4.4 LTS and adds a Docker development environment.

Resolves: #3.

Commit:
Upgrade Symfony 3.1 to 3.4

This upgrades Symfony 3.1 to minor new version 3.4 in order to prepare to upgrade to version 4.0.

See: #3.

Commit:
Add Docker development environment

This adds a docker-compose.yml that contains the three necessary services to develop on the project (MySQL, phpMyAdmin and PHP-FPM 7.2).
It also contains a Blackfire image to profile the application's performance.

PHP-FPM 7.2 image is build from Dockerfile and contains the Blackfire PHP probe.

Commit:
Fix Symfony deprecations

This fixes Symfony deprecations in order to migrate to 4.4.

See: #3.

Commit:
Migrate from Symfony 3.4 to 4.4

This upgrades Symfony 3.4 to 4.4 LTS.
File structure has been modified in accordance with Symfony Flex.
Dockerfile image for PHP-FPM has been changed to upgrade PHP from 7.2 to 7.4 to match Symfony 4.4 requirements.

See: #3.

PierreSylvain commented:
**Attention** le fichier **.env.local** est dans le repository alors qu'il ne devrait pas y être.
Aucun problème d'installation détecté.
Migration réussie

briandidierjean commented:
> **Attention** le fichier **.env.local** est dans le repository alors qu'il ne devrait pas y être.
> Aucun problème d'installation détecté.
> Migration réussie

Bien noté, je corrige ça. Merci Pierre.

Commit:
Add Docker database credentials to .env file

This adds Docker database credentials to .env file and removes the .env.local file as it should not be tracked.
